src/openms/CMakeLists.txt

```diff
@@ -35,8 +35,7 @@
 project("OpenMS")
 cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
 
-set(CMAKE_AUTOUIC ON)
-set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTOMOC ON) # some OpenMS classes make use of Signal/Slot, e.g. SYSTEM/FileWatcher.cpp
```

Comment:
why not autouic? maybe add comment why this is not a good idea

Reply:
I remove it, because if someone adds UIs to OpenMS and not to OpenMSGUI, this deserves an error.